app/locale.js

```diff
@@ -258,7 +258,257 @@ var rendererIdentifiers = function () {
     'autoplayMedia',
     // Release channels
     'channelDev',
-    'channelBeta'
+    'channelBeta',
+    // Country Codes,
+    'AF',
```

Comment:
Since this list is also stored in the countries.js, can we import that instead (and then append or merge the arrays)?  This way, we're only keeping track of the list in one place 😄 👍